src/components/ButtonsToolbar/index.js

```diff
@@ -29,9 +30,9 @@ const ButtonsToolbar = ({
         <Toolbar modifiers={['mobile-reverse-order', 'bottom']}>
           <ToolbarList>
             {onNavigatePrevPage && (
-              <UtrechtLinkButton onClick={onNavigatePrevPage} appearance="subtle-button">
+              <Anchor href={previousPage} onClick={onNavigatePrevPage}>
```

Comment:
I think we can use our [`Link`](https://github.com/open-formulieren/open-forms-sdk/blob/main/src/components/Link.js#L9) component and drop the `onNavigatePrevPage` entirely, but I haven't verified that